ffi/derive/tests/ui_fail/mutable_input_arg.rs

```diff
@@ -1,30 +0,0 @@
-//! Test which assures that methods taking mutable references as input arguments are not
```

Comment:
So now we accepts mutable input args? What it is done for and why it wasn't like that before?

Reply:
yeah, it's not used anywhere except for input mutable handles. Previously, it was a nuisance to support this

Reply:
While being more general is good IMO, since we're not using it, you could also leave it as is.

Reply:
I wanted to leave it as is, but it ended up being complicated because now all types go through this conversion and handle type can be received as `&mut Self`. So API got expanded for free